netflix/src/components/home/Today.tsx

```diff
@@ -0,0 +1,48 @@
+import Image from 'next/image';
+import { getTopRatedMovie } from '@/api/getMovies';
+import TopNImg from '@/public/icon/topN.svg';
+import PlusImg from '@/public/icon/plus.svg';
+import PlayBtn from '@/components/common/PlayBtn';
+import InfoImg from '@/public/icon/info.svg';
+
+export default async function Today() {
+  const moviesData = await getTopRatedMovie();
+  const randomIndex = Math.floor(Math.random() * moviesData.length); // 랜덤 인덱스 생성
+  const randomMovie = moviesData[randomIndex]; // 랜덤 영화 선택
+
+  return (
+    <div>
+      <div className="w-full h-[415px] relative">
+        <div className="absolute inset-0">
+          {randomMovie && (
+            <Image
+              fill
+              src={`https://image.tmdb.org/t/p/original${randomMovie.poster_path}`}
+              alt={randomMovie.title}
+              style={{ objectFit: 'cover' }}
+              priority
+            />
+          )}
+        </div>
+        <div className="w-full h-full absolute bg-gradient-to-b from-transparent via-transparent to-black"></div>
+      </div>
+      <section className="flex flex-row justify-center gap-[5px] pt-0.5">
+        <TopNImg />
+        {randomIndex !== null && (
+          <div className="fonts-today">Top {randomIndex + 1} in Korea Today</div>
+        )}
+      </section>
+      <section className="flex justify-between w-full h-[45px] mt-[11px] pl-[54px] pr-[62px]">
+        <div className="flex flex-col items-center">
+          <PlusImg />
+          <div className="fonts-mainicon">My List</div>
+        </div>
+        <PlayBtn width={6.91406} />
+        <div className="flex flex-col items-center">
+          <InfoImg />
+          <div className="fonts-mainicon">Info</div>
+        </div>
+      </section>
+    </div>
+  );
+}
```

Comment:
Today.tsx 컴포넌트에서 이미지 띄우기 / ...in Korea Today 텍스트 띄우기 /버튼 바 띄우기 이렇게 여러 성격의 기능들이 들어가 있어서 컴포넌트를 좀만 더 생성해줘서 세분화해주면 더 가독성에 좋을 것 같아요~!!

```suggestion
return <div>
   <BigMovieImage />   -> 이미지 띄우기
   <Text text={`Top ${randomIndex + 1} in Korea Today`} className="fonts-today" /> -> ...in Korea Today 텍스트  
   <ButtonBar /> -> MyList, Play, info 을 가지고 있는 버튼 바
</div>
```


( in Korea Today 부분은 이미 짜주신 것 처럼 div 태그에 해주셔도 상관 없는데, 저는 개인적으로 Text 컴포넌트를 하나 파서 저런식으로 넘겨주면 가독성 측면에서 더 좋아지는 거 같더라구여 ㅎㅎ)

Text 컴포넌트 예시(twMerge 라이브러리 사용)  ->
```
   return <p className={twMerge("초기 설정해 준 style", className)}>
      {text}
   </p>
```